shared/actions/pinentry.js

```diff
@@ -28,8 +27,28 @@ export function registerPinentryListener (): AsyncAction {
       payload: {started: true},
     }: RegisterPinentryListenerAction))
 
-    const pinentryListeners = pinentryListenersCreator(dispatch)
-    engine.listenGeneralIncomingRpc(pinentryListeners)
+    engine.setIncomingHandler('keybase.1.secretUi.getPassphrase', (payload, response) => {
```

Comment:
we don't really do this elsewhere. also with the current stack you can see its a getPassphrase call and thats its a handler so i think its all easy to understand